app.rb

```diff
@@ -0,0 +1,22 @@
+require "bundler/setup" # require all the gems we'll be using for this app from the Gemfile. Obviates the need for `bundle exec`
+require "pg" # postrgres db library
+require "active_record" # the ORM
+require "pry" # for debugging
+require 'sinatra'
+require 'sinatra/reloader'
+
+# Connect to DB
+require_relative "db/connection"
+
+# Load specific routes / controllers
+require_relative "controllers/apartments.rb"
+require_relative "controllers/tenant.rb"
```

Comment:
the `.rb` is unecessary here, because you can only `require_relative` ruby files.